src/document.ts

```diff
@@ -0,0 +1,3 @@
+export interface Document {
```

Comment:
maybe we can just move this to `bson.ts`? I'm loathe to add even more files to this project..